WhatsOnTap/Controllers/BarsController.cs

```diff
@@ -32,6 +33,90 @@ public ActionResult Details(int id)
             return View(viewModel);
         }
 
+        [HttpGet("/bars/{id}/edit")]
+        public ActionResult Edit(int id)
+        {
+            BarDetailsViewModel viewModel = new BarDetailsViewModel(_db, id);
+            viewModel.FindAllBeers();
+            return View(viewModel);
+        }
+
+        [HttpPost("/bars/{id}/edit")]
+        public ActionResult Edit(List<int> BeerId, string barName, string barRating, string barWebsite, string barStreet, string barCity, string barState, string barZip, string barPhone, string barLatitude, string barLongitude, int id)
```

Comment:
I think we can package these properties up into an object of the class similar to what is happening here: https://github.com/atrotter0/snappy-snips/blob/master/HairSalon/Controllers/StylistsController.cs#L35. This way we do not need to set each property or pass each property through individually. As long as it is every property on the class, we can package it up and pass it through.